### PR TITLE
Tsneez visualization

### PR DESCRIPTION
**Pull request by samirrajpurkar**
adds tsneez visualization as example 3 in the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,8 +58,204 @@ document.addEventListener('DOMContentLoaded', function() {
...
+    fetch(shortgloveFile)
+        .then(data => data.json())
+        .then((vecs) => {
+            dimensionReduce(vecs)
+    });
+});
+
+// Set up visualization
+var svg
+var fadeOld = 0
```
Comment (rajpurkar) [inline README.md:176]: can remove

File: README.md [hunk 2]
```diff
@@ -58,8 +58,204 @@ document.addEventListener('DOMContentLoaded', function() {
...
+        } else {
+        d.rotate = d.rotate
+        }
+        return 'translate(' +
+            ((Y.get(i, 0) * 200 * ss + tx) + 400) + ',' +
+            ((Y.get(i, 1) * 200 * ss + ty) + 400) + ')' +
+            'rotate(' + d.rotate + ')'
+    })
+
+    s.selectAll('rect').style('fill-opacity', function (d) {
```
Comment (rajpurkar) [inline README.md:251]: everything below thing line can be removed. deals with addition of new points

File: README.md [hunk 3]
```diff
@@ -58,8 +58,204 @@ document.addEventListener('DOMContentLoaded', function() {
...
+
+    let converted = convertToD3Format(vecs)
+    drawEmbedding(converted)
+    window.requestAnimationFrame(animate) 
+}
+
+
+
+function dimensionReduce(vecs) {
+    var NUMBER_OF_EMBEDDINGS = 1000
```
Comment (rajpurkar) [inline README.md:159]: would pull this out to be defined at top of page

File: README.md [hunk 4]
```diff
@@ -58,8 +58,204 @@ document.addEventListener('DOMContentLoaded', function() {
...
+                display: block;
+            }
+            .viewport svg text {
+                pointer-events: none;
+                font-family: 'Lato', Arial;
+                font-weight: 500;
+                font-size: 1em;
+                fill: #444;
+            }
+            .infoCard {
```
Comment (rajpurkar) [inline README.md:94]: Info card and info body are not being used. remove!

File: README.md [hunk 5]
```diff
@@ -58,8 +58,204 @@ document.addEventListener('DOMContentLoaded', function() {
...
+                position: absolute;
+                top: 0px;
+            }
+        </style>
+        
+    </head>
+    <body>
+        <div class="container">
+            <div class="row">
+                <div class="infoCard">
```
Comment (rajpurkar) [inline README.md:107]: no need for infoCard

**Review by rajpurkar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (samirrajpurkar) [reply README.md:176]: We need this as these are global variables.

File: README.md (hunk 2 above)
Comment (samirrajpurkar) [reply README.md:251]: ok and hence var fadeOld varaible also removed

File: README.md (hunk 3 above)
Comment (samirrajpurkar) [reply README.md:159]: Ok. done.

File: README.md (hunk 5 above)
Comment (samirrajpurkar) [reply README.md:107]: Ok. 
Added style for "container" class.

**Comment by samirrajpurkar** [comment]
All changes are done.

**Pull request closed without merging**